bindings/python/iree/compiler/debugging.py

```diff
@@ -0,0 +1,172 @@
+"""Debugging support."""
+
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Swap the module doc and the license header


```suggestion
# Copyright 2021 The IREE Authors
#
# Licensed under the Apache License v2.0 with LLVM Exceptions.
# See https://llvm.org/LICENSE.txt for license information.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

"""Debugging support."""
```

Reply:
Yes, basically. We should not be passing file paths down across boundaries - the explicitness is what makes each way that someone uses this a snowflake with its own mechanism for getting at the vital debug information (i.e. it's broken and I could fix it if I had just a bit more information, but first I have to go grok/hack whatever project is using it to enable the ability to get more information). This is pretty standard fare for Python.